parsl/serialize/base.py

```diff
@@ -0,0 +1,68 @@
+from abc import ABCMeta, abstractmethod
+import logging
+import functools
+
+logger = logging.getLogger(__name__)
+
+# GLOBALS
+METHODS_MAP_CODE = {}
+METHODS_MAP_DATA = {}
+
+
+class fxPicker_enforcer(metaclass=ABCMeta):
```

Comment:
give this and fxPicker_shared a better a name?

Reply:
actually fxPicker_enforcer is not used, i think, so remove

Reply:
and move abstractmethods into fxPicker_shared

Reply:
Done, and it's now renamed to `SerializerBase`